README.md

```diff
@@ -462,13 +460,62 @@ Additionally, the OIDC `redirect_uri` parameter must be set to the HTTP API's au
 This is required, as the Authorization Code Grant needs to be redirected back to the HTTP API, to complete
 HTTP API token generation.
 
-### Add your specified `irods_user_claim` to the user's claims
+### Configure your 'user_mapping'
 
-Currently, the server looks for the custom claim in the ID Token, which is specified in the `irods_user_claim` parameter.
-This serves as the mapping mechanism for an OIDC User to an iRODS User.
+Before you can use OpenID with iRODS, you must enable one of the two shipped plugins, or develop your own to suit your specific needs.
 
-A user who authenticates but does not have the claim specified in `irods_user_claim` mapped in their account
-will not have access to the API. A HTTP 400 Bad Request status code will be returned if the claim specified in `irods_user_claim` is not found.
+#### Local File plugin
+
+To use this plugin, make sure to load `libirods_http_plugin-local_file.so`.
```

Comment:
Hmm, shouldn't `libirods_http_plugin` be `libirods_http_api`?

Reply:
This is how it is.

https://github.com/irods/irods_client_http_api/blob/e977b247714d7b244c7dbe6a98dab1c2dd1585bd/plugins/user_mapping/CMakeLists.txt#L9-L10

We can change these if we wish to rename it.

Reply:
Yes please include `api` before `plugin`.

Reply:
Done.